tests/data/expected/main/main_openapi_const_field_msgspec/output.py

```diff
@@ -4,7 +4,7 @@
 
 from __future__ import annotations
 
-from msgspec import Struct
+from msgspec import Meta, Struct
```

Comment:
@indrat 
I think the PR fixes the issue too https://github.com/koxudaxi/datamodel-code-generator/issues/1607

Reply:
This does fix the missing `constr` import and does appear to fix a similar issue for the msgspec output but not #1607 specifically. #1607 *isn't* using `--collapse-root-models` so is ending up with the root model in `common.py` without the `Meta` import.

Reply:
Thank you for checking the pattern.
OK, I will fix the issue in another PR.